klpbuild/plugins/setup.py

```diff
@@ -10,6 +10,7 @@
 from klpbuild.klplib import utils
 from klpbuild.klplib.cmd import add_arg_lp_name, add_arg_lp_filter
 from klpbuild.klplib.codestreams_data import get_codestreams_data, set_codestreams_data, store_codestreams
+from klpbuild.klplib.supported import get_supported_codestreams
```

Comment:
The idea of no_check was introduced when we planned to support creating livepatch for any function in the kernel. Since that ship sailed a long time ago, and we only care about CVEs, and we expect them to be backported before creating the LP, I would say that we can also drop it from setup. But we can do that in a different PR.